piximi-documentation/measurements.md

```diff
@@ -4,4 +4,104 @@ While sometimes, the goal of an experiment is just to find objects or to classif
+scale: 50%
+---
+The circular indicator next to `Tables` indicates how far through measurement generation Piximi is. Make sure to keep the Piximi window open (consider moving it into its own tab) to let calculations continue!
+```
+
+5. Once the initial pre-calculations are done, you can select which measurements you'd like to make, as well as how you'd like to subset "split" the data (currently by `Category` or by `Partition`; if you only have one category, tha's fine, but you do have to click something here!). As before, this will take some time, so just leave the Piximi tab open and the circular indicator will let you know how long this is going to take.
+
+```{figure} ./img/measurement-walkthrough/measure-03-select-splits-and-measurements.png
+---
+name: measure-03-select-splits-and-measurements
+scale: 50%
+---
+Select the split or splits you want to measure for each table, as well as the measurements you want for that table. Just like during table creation, a circular indicator next to the word `Measurements` will tell you how far along Piximi is at generating your measurements.
+```
+
+6. Once your measurements have been generated, they will appear in the main area in a data grid! In the grid, you can see the measurement name, the subset (or `Split`) of the data represented, and the Mean, Median, and Standard deviation of the data for that feature in that subset.
+
+```{figure} ./img/measurement-walkthrough/measure-04-data-grid.png
+---
+name: measure-04-data-grid
+---
+The Piximi data grid
+```
+
+7. If you'd like to plot data in Piximi, simply navigate to the data grid containing the data you want to plot and hit the Plots tab at the top. You can make as many plots as you would like by hitting the `+` button on the plots tab. Plots can also be exported by saving to PNG. Piximi includes several plot types, including scatter plots, swarm plots, and histograms.
+
+```{figure} ./img/measurement-walkthrough/measure-05-scatter.png
+---
+name: measure-05-scatter
+scale: 50%
+---
+Scatter plots generated in Piximi can have X, Y, and Size measurements selected; they can also show colors according to the selected splits (with many colormaps to choose from). Note the "Save to PNG" button in the bottom right!
+```
+
+```{figure}  ./img/measurement-walkthrough/measure-06a-swarm-no-category.png
+---
+name: measure-06a-swarms
+scale: 50%
+---
+Swarm plots can be shown on their own...
+```
+
+```{figure} ./img/measurement-walkthrough/measure-06b-swarm-with-category.png
+---
+name: measure-06b-swarms
+scale: 50%
+---
+... or with a summary boxplot overlaid.
+```
+
+```{figure} ./img/measurement-walkthrough/measure-07-histogram.png
+---
+name: measure-07-histogram
+scale: 50%
+---
+You can choose to represent your data as a histogram, with a custom number of bins.
+```
+
+If you want to do further analyses of your data (and you should)!, you can export your images from the measurements menu  - this will let you do whatever downstream analysis you like. 
```

Comment:
should this be "export your *measurements* from the measurements menu" ?

Reply:
Yes, yes it should